Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -668,4 +674,32 @@ def test_search
   ensure
     desc_cache.unlink
   end
+
+  def test_bundle
+    skip_unless_test_cmd_taps
+    HOMEBREW_REPOSITORY.cd do
+      shutup do
+        system "git", "init"
+        system "git", "commit", "--allow-empty", "-m", "This is a test commit"
+      end
+    end
+
+    mktmpdir do
+      FileUtils.touch "Brewfile"
+      assert_equal "The Brewfile's dependencies are satisfied.",
+        cmd("bundle", "check")
+    end
+  ensure
+    (HOMEBREW_REPOSITORY/".git").rmtree
```

Comment:
@UniqMartin 👍. I think we should probably be using `FileUtils.rm_rf` for all these cases to avoid the failure when it doesn't exist and avoid guarding them all with `.exist?`